show/main.py

```diff
@@ -410,6 +405,43 @@ def interfaces():
     """Show details of the network interfaces"""
     pass
 
+# 'alias' subcommand ("show interfaces alias")
+@interfaces.command()
+@click.argument('interfacename', required=False)
+def alias(interfacename):
```

Comment:
This command was intentionally removed. Did you mean to add it back, or was your code out-of-date?

Reply:
My bad. I thought I had grabbed the most recent version of the file but it looks like I missed a step.   I've updated this now.